src/components/WhatsNew.vue

```diff
@@ -0,0 +1,133 @@
+<template>
+<div id="viz-latest">
+    <div id="viz-title">
+        <h2 >What's new  </h2><h3 id="title-new"><a href="https://twitter.com/USGS_DataSci/status/1431332579634991114/photo/1" target="_blank">Data assimilation animated</a> </h3>
```

Comment:
Wondering how it would look to have the 'Data assimilation animated' left justified beneath 'What's new'

Reply:
![image](https://user-images.githubusercontent.com/17803537/136105318-91feefbd-8856-495d-8bd0-a69e219c1125.png)

I like that. It gives more space to the right side and could add more details if we wanted.